src/Symfony/Component/Mailer/Transport/Smtp/EsmtpTransport.php

```diff
@@ -61,6 +62,10 @@ public function __construct(string $host = 'localhost', int $port = 0, bool $tls
 
         $stream->setHost($host);
         $stream->setPort($port);
+
+        if (false === $verifyPeer) {
+            $this->verifyPeer = false;
```

Comment:
there is no need for a new property, instead, the parent::__construct() method should be passed a new SocketStream() with stream socket options properly configured

Reply:
I configure the stream directly like you suggested. But i'm not sure if I've used the right way to do it, because my `SocketStream::streamContextOptions` could be set later if `SocketStream::setStreamOptions` is called before the `SocketStream::initialize` function is called

Reply:
this looks good to me